### ISSUE TITLE
Can't clear pathfinders from wandering trader on 1.16.5

### ISSUE THREAD

**Issue by datatags** [GamerCoder215/MobChip#76]
### Bug Type

Compatibility Issues, Console Errors

### MobChip Version

1.9.0

### Server Version

Spigot 1.16.5

### Description

Clearing pathfinders from a wandering trader produces an error. This is unrelated to #74 and #75. The error can only be observed on Spigot, I suspect due to #75.

### Expected Behavior

Pathfinders should be cleared

### Actual Behavior

An exception is thrown: (Root exception is a CommandException so I have omitted it)
```
Caused by: java.lang.AssertionError: Could not convert org.bukkit.entity.Illusioner to NMS class
        at be.isach.ultracosmetics.shaded.mobchip.abstraction.v1_16_R3.ChipUtil1_16_R3.toNMS(ChipUtil1_16_R3.java:155) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.abstraction.v1_16_R3.ChipUtil1_16_R3.toNMS(ChipUtil1_16_R3.java:184) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.abstraction.v1_16_R3.ChipUtil1_16_R3.removePathfinder(ChipUtil1_16_R3.java:389) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil.lambda$clearPathfinders$0(ChipUtil.java:54) ~[?:?]
        at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.abstraction.ChipUtil.clearPathfinders(ChipUtil.java:54) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.bukkit.BukkitAI.updateAI(BukkitAI.java:42) ~[?:?]
        at be.isach.ultracosmetics.shaded.mobchip.bukkit.BukkitAI.clear(BukkitAI.java:164) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.pets.Pet.clearPathfinders(Pet.java:138) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.pets.Pet.initializeEntity(Pet.java:122) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.pets.Pet.onEquip(Pet.java:87) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.Cosmetic.equip(Cosmetic.java:105) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.type.CosmeticType.equip(CosmeticType.java:157) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.type.PetType.equip(PetType.java:66) ~[?:?]
        at be.isach.ultracosmetics.cosmetics.type.PetType.equip(PetType.java:31) ~[?:?]
        at be.isach.ultracosmetics.command.subcommands.SubCommandToggle.toggle(SubCommandToggle.java:92) ~[?:?]
        at be.isach.ultracosmetics.command.subcommands.SubCommandToggle.onExePlayer(SubCommandToggle.java:48) ~[?:?]
        at be.isach.ultracosmetics.command.CommandManager.onCommand(CommandManager.java:81) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        ... 19 more
```
To confirm, yes, the error says Illusioner when trying to clear pathfinders from a Wandering Trader.

**Comment by GamerCoder215** [comment]
The Illusioner class is missing from the NMS Mapper, so it is part of a separate bug.
> To confirm, yes, the error says Illusioner when trying to clear pathfinders from a Wandering Trader.

Are you sure? The Wandering Trader Class is actually there, and the only way this error actually pops up is if all of these fail:
- The Class is in the Bukkit-to-NMS Class Map (Wandering Trader is, Illusioner is not)
- The NMS Class Name matches the name of the Bukkit Class (Illusioner: `EntityIllagerIllusioner`)
- The NMS Class Name matches the name of the Bukkit Class with `Entity` appended (Illusioner: `EntityIllagerIllusioner`)

**Comment by GamerCoder215** [comment]
Fixed in d745f9e. Let me know if the issue persists due to the wandering trader issue.
